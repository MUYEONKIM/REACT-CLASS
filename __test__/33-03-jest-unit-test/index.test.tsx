import JestUnitTestPage from "../../pages/section33/33-03-jest-unit-test";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

it("기존이랑 바뀐게 없는지 비교해보자!! - 스냅샷 테스트", () => {
    // const result = screen.getByText("철수는 13살 입니다.");
    // expect(result).toBeInTheDocument();

    // const result2 = screen.getByText("취미 :");
    // expect(result2).toBeInTheDocument();

    // const result3 = screen.getByText("철수랑 놀러가기");
    // expect(result3).toBeInTheDocument();

    const result = render(<JestUnitTestPage />);
    expect(result.container).toMatchSnapshot();
});
