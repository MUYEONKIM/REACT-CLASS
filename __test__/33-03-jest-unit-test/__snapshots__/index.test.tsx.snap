// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존이랑 바뀐게 없는지 비교해보자!! - 스냅샷 테스트 1`] = `
<div>
  <div>
    철수는 14살 입니다.
  </div>
  취미 : 
  <input
    type="test"
  />
  <button>
    철수랑 놀러가기
  </button>
</div>
`;
